workflow/controller/controller.go

```diff
@@ -534,6 +534,65 @@ func (wfc *WorkflowController) signalContainers(namespace string, podName string
 	return time.Duration(*pod.Spec.TerminationGracePeriodSeconds) * time.Second, nil
 }
 
+func (wfc *WorkflowController) deleteOffloadedNodes() error {
```

Comment:
can you move this func below workflowGarbageCollector so there are less risk of Git conflicts?